src/app/defaultIntlAdapter.ts

```diff
@@ -0,0 +1,18 @@
+import {IntlAdapter} from "./commonTypes";
```

Comment:
maybe let's add some comment to explain the necessary of default adapter, because w/o context I think it looks strange